src/Search.java

```diff
@@ -0,0 +1,90 @@
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class Search {
+	static Scanner scanner=new Scanner(System.in);
+	public static void main(String[] args)
+	{
+		System.out.println("Enter the size of array");
+		int temp=getInput();
+		if(temp<=0)
+		{
+			scanner.nextLine();
+			temp=getInput();
+		}
+		int arr[]=new int[temp];
+		for(int loop_var=0;loop_var<temp;loop_var++)
+		{
+			System.out.print("Enter element "+(loop_var+1)+" = ");
+			arr[loop_var]=getInput();
+		}
+		System.out.println("Enter the element to search");
+		int to_search=getInput();
+		Search search=new Search();
+		System.out.println("Index of "+to_search+" = "+search.binarySearch(arr,0,arr.length,to_search)); //It will only work for sorted array
```

Comment:
Provide an option to user that which search he want's to perform.

Reply:
done